src/lib/stitching/causality/stitching.ts

```diff
@@ -53,6 +86,19 @@ export const causalityStitchingEnvironment = ({
     `,
 
     resolvers: {
+      Lot: {
+        internalID: {
+          resolve: ({ saleArtwork }) => saleArtwork._id,
+        },
+        saleArtwork: {
```

Comment:
#minor you probably don't need this and the `lot` resolver (the default one when not provided is to resolve the same-named key, so you're sort of replicating the default resolver)